lib/screens/annonce/annonce_list.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter/material.dart';
+
+class AnnonceList extends StatefulWidget {
+  const AnnonceList({Key? key}) : super(key: key);
+
+  @override
+  _AnnonceListState createState() => _AnnonceListState();
+}
+
+class _AnnonceListState extends State<AnnonceList> {
+  Stream<QuerySnapshot> _annonces =
+      FirebaseFirestore.instance.collection('annonces').snapshots();
+
+  String bro = "BRO";
+
+  @override
+  Widget build(BuildContext context) {
+    return StreamBuilder<QuerySnapshot>(
+        stream: _annonces,
+        builder: (context, snapshot) {
+          if (snapshot.hasError) {
+            return Text("Error !");
+          }
+
+          if (snapshot.connectionState == ConnectionState.waiting) {
+            return Center(
+              child: CircularProgressIndicator(
+                color: Colors.teal,
+              ),
+            );
+          }
+
+          // return Column(
+          //   mainAxisAlignment: MainAxisAlignment.center,
+          //   mainAxisSize: MainAxisSize.max,
+          //   children: [
+          //     Icon(Icons.flight_takeoff),
+          //     Text("Annonces"),
+          //   ],
+          // );
```

Comment:
I think it's better to remove all anused code not to let it hear  commented

Reply:
Done ✔️